homeassistant/components/light/osramlightify.py

```diff
@@ -195,7 +195,7 @@ def turn_on(self, **kwargs):
             self._brightness = kwargs[ATTR_BRIGHTNESS]
             _LOGGER.debug("turn_on requested brightness for light: %s is: %s ",
                           self._name, self._brightness)
-            self._brightness = self._luminary.set_luminance(
+            self._luminary.set_luminance(
```

Comment:
You have included an unrelated commit. It's so tiny that it does not matter much this time around, but please make sure to start a fresh branch from `dev` when you create a new PR.